terraform/environments/bootstrap/secure-baselines/README.md

```diff
@@ -1,6 +1,13 @@
 # Modernisation Platform: environments bootstrapping
```

Comment:
I think this README might have been intended for the parent directory (`terraform/environments/bootstrap`)

Reply:
Agreed. Feel free to move it!